src/Microsoft.DotNet.Cli.Utils/project.json

```diff
@@ -2,7 +2,13 @@
   "version": "1.0.0-preview3-*",
   "buildOptions": {
     "keyFile": "../../tools/Key.snk",
-    "warningsAsErrors": true
+    "warningsAsErrors": true,
+    "compile": {
+      "include": [
+        "**/*.cs",
+        "../../src/dotnet/DotnetFiles.cs"
```

Comment:
I don't feel this is a good idea.  I explicitly just moved this file out of this library.

This library can be used outside of the CLI, so it has no idea how to get to the .version file (or any of the CLI's "files").  We need to find a different way of doing what you are trying to do.